contrib/dist/release_checklist_issue_template

```diff
@@ -0,0 +1,102 @@
+## Make a perfect tarball
```

Comment:
Should probably name this file with .md.

Note that I typically update this checklist with every release, too.  Where is the right place to maintain this?  I.e., is it enough to go find the issue corresponding to the last release checklist and use that?  Or should we maintain / update a list here in `contrib/dist`?